### PR TITLE
✨ SWG Encryption Client Implementation Change

### PR DESCRIPTION
**Pull request by chenshay**
Changing how SwG Encryption should be written on the client side
@ampproject/wg-access-subscriptions
#21658

### CODE REVIEW

File: extensions/amp-subscriptions/validator-amp-subscriptions.protoascii [hunk 1]
```diff
@@ -68,4 +68,33 @@ tags: {  # amp-subscriptions-google
   requires_extension: "amp-subscriptions"
   attr_lists: "common-extension-attrs"
 }
+tags: {  # script
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "script tag that contains encrypted material"
+  mandatory_parent: "CONTENT"
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/validator-amp-subscriptions.protoascii]: What's the "CONTENT" parent? Sorry, not super familiar with validator rules. Is it an actual tag name? We don't have "CONTENT" tags anywhere that I know of...

Comment (jpettitt) [reply extensions/amp-subscriptions/validator-amp-subscriptions.protoascii]: I think that's meant to be a parent with a `subscriptions-section="content"` attribute but I'm not sure how that is represented in validator

Comment (dvoytenko) [reply extensions/amp-subscriptions/validator-amp-subscriptions.protoascii]: That's what I'd expect, but it doesn't look like validator rules currently reflect that.

Comment (chenshay) [reply extensions/amp-subscriptions/validator-amp-subscriptions.protoascii]: resolved offline

Comment (chenshay) [reply extensions/amp-subscriptions/validator-amp-subscriptions.protoascii]: resolved offline

**Pull request merged**